libvmi/driver/kvm/kvm.c

```diff
@@ -1364,6 +1380,17 @@ kvm_init_vmi(
     }
     vmi->num_vcpus = info.nrVirtCpu;
 
+#ifndef HAVE_LIBVMI_REQUEST
+    char *version = exec_info_version(kvm);
+    dbprint(VMI_DEBUG_KVM, "--Checking QEMU version string : %s", version);
+    // find substring "vmi"
```

Comment:
So there is no point appending "vmi" to the version string. The version should be checked whether it is 2.8 or later and if so, error out.